.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Let's also add:
```yaml
  pull_request:
    branches:
      - main
```
to ensure PRs don't break the build. We don't need live previews, but just building will be helpful.